airflow/providers/amazon/aws/operators/ecs.py

```diff
@@ -144,7 +144,7 @@ class ECSOperator(BaseOperator):
 
     ui_color = '#f0ede4'
     template_fields = ('overrides',)
-    template_fields_renderers = {"overrides": "py"}
+    template_fields_renderers = {"overrides": "json"}
```

Comment:
Overrides is a dict no a json , no ?

Reply:
@kaxil yes, it's a dict, but then `py` renderer says that there is no source code for dict class. So i thought `json` fix it

Reply:
Can you post a before/after screenshot, please?

Reply:
It looks really good. Yes, it might be a bit confusing since it's not really json, but it formats it correctly.

![image](https://user-images.githubusercontent.com/2885779/125200541-ee296380-e26b-11eb-8508-6912a29142de.png)

You can check the before screenshot in the issue.

Reply:
Awesome, thanks. Sounds good